java/sage/VideoFrame.java

```diff
@@ -5049,12 +5050,16 @@ else if (QTMediaPlayer.canLoadMediaFile(theFile))
 
   public MediaPlayer createMediaPlayer(MediaFile theFile)
   {
+      
```

Comment:
Please remove all the extra blank lines and debug statements you added to this file.